internal/pkg/describe/env.go

```diff
@@ -274,6 +302,15 @@ func (e *EnvDescription) HumanString() string {
 		fmt.Fprintf(writer, "  %s\t%s\n", svc.Name, svc.Type)
 	}
 	writer.Flush()
+	fmt.Fprint(writer, color.Bold.Sprint("\nJobs\n\n"))
```

Comment:
Do you think we want to only show the `Jobs` section only when there is/are job(s) deployed (btw I think what you have now is fine too)

Reply:
I guess it is okay for the awareness of the customers that Jobs are also being showed as a part of `env show`. And also it aligns with the current implementation for services.